PSKoans/Koans/Katas/SortingCharacters.Koans.ps1

```diff
@@ -67,23 +67,23 @@ Describe 'Kata - Sorting Characters' {
         }
         @{
             String = 'What do you call the world?'
-            Result = 'aacddehhlllooorttuwWy'
+            Result = 'aacddehhlllooorttuWwy'
         }
         @{
             String = 'Out of nowhere, the mind comes forth.'
-            Result = 'cdeeeeffhhhimmnnooOoorrstttuw'
+            Result = 'cdeeeeffhhhimmnnOoooorrstttuw'
         }
         @{
             String = 'Because it is so very clear, it takes longer to come to the realization.'
             Result = 'aaaaaBccceeeeeeeeeghiiiiiklllmnnoooooorrrrsssstttttttuvyz'
         }
         @{
             String = 'The hands of the world are open.'
-            Result = 'aaddeeeefhhhlnnoooprrstTw'
+            Result = 'aaddeeeefhhhlnnoooprrsTtw'
         }
         @{
             String = 'You are those huge waves sweeping everything before them, swallowing all in their path.'
-            Result = 'aaaaabeeeeeeeeeeeefgggghhhhhhiiiiillllmnnnnoooopprrrrsssstttttuuvvwwwwyY'
+            Result = 'aaaaabeeeeeeeeeeeefgggghhhhhhiiiiillllmnnnnoooopprrrrsssstttttuuvvwwwwYy'
```

Comment:
Can you share the code you used to get these results?

I remember what I did to get the result I laid out in there, but there's a good chance it didn't play nice with the upper and lower cases. This looks right, just want to verify :)